content/en/docs/armory-admin/dinghy-enable.md

```diff
@@ -3,12 +3,18 @@ title: Enabling Pipelines as Code
 aliases:
   - /spinnaker/install_dinghy/
   - /docs/spinnaker/install-dinghy/
+description: >
+  Pipelines as Code allows you to create and maintain pipeline templates in source control.
 ---
 
+{{< include "note-github-branch-name.md" >}}
+
 This guide includes:
 
 * Configurations for enabling Armory's Pipelines as code feature using Spinnaker Operator or Halyard
 * Settings for GitHub, GitLab, or Bitbucket/Stash webhooks to work with the Pipelines as code
+  * If you use GitHub, see [Custom branch configuration]() for information about how to explicitly set the branch that Pipelines as Code uses.
```

Comment:
`If you use GitHub, see [Custom branch configuration](#custom-branch-configuration) for information about how to explicitly set the branch that Pipelines as Code uses.`